### PR TITLE
TCOMP-558 jdbc drivers

### PR DESCRIPTION
**Pull request by gonchar-ivan**
**What is the current behavior?** (You can also link to an open issue here)

derby and mysql drivers

**What is the new behavior?**

6 more drivers
see branch https://github.com/Talend/components/tree/igonchar/TCOMP-558_it
those branch is not very convenient, but each commit contains test for specific db

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://jira.talendforge.org/browse/TCOMP-558

### CODE REVIEW

File: services/components-maven-repo/config/jdbc_config.json [hunk 1]
```diff
@@ -2,22 +2,227 @@
...
+		"url" : "jdbc:jtds:sqlserver://localhost:1433/db",
+		"paths" : 
+	 	[
+	 		{"path" : "mvn:net.sourceforge.jtds/jtds/1.3.1"}
+	 	]
+	
+	},
+	
+	{
+		"id" : "AZURE_SQL",
```
Comment (gonchar-ivan) [inline services/components-maven-repo/config/jdbc_config.json]: SQL_SERVER AND AZURE_SQL have the same configuration.
I'm not sure whether we need to different id

File: services/components-maven-repo/config/jdbc_config.json [hunk 2]
```diff
@@ -2,22 +2,227 @@
...
+			{"path" : "mvn:org.apache.httpcomponents/httpcore/4.4"},
+			{"path" : "mvn:org.apache.thrift/libthrift/0.9.3"},
+			{"path" : "mvn:org.apache.zookeeper/zookeeper/3.4.6"},
+			{"path" : "mvn:org.slf4j/slf4j-log4j12/1.6.1"},
+			{"path" : "mvn:log4j/log4j/1.2.16"},
+			{"path" : "mvn:jline/jline/0.9.94"},
+			{"path" : "mvn:io.netty/netty/3.7.0.Final"},
+			{"path" : "mvn:org.apache.curator/curator-framework/2.6.0"},
+			{"path" : "mvn:org.apache.curator/curator-client/2.6.0"},
+			{"path" : "mvn:org.slf4j/slf4j-api/1.7.5"}
```
Comment (gonchar-ivan) [inline services/components-maven-repo/config/jdbc_config.json]: Here we have a problem with jdk tools

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/551/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/551/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/562/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/562/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/566/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/566/)

**Pull request merged**